pkg/cli/config/config_test.go

```diff
@@ -26,6 +26,19 @@ func TestGetBackplaneConfig(t *testing.T) {
 		}
 	})
 
+	t.Run("It should fail if the proxy-url is not configured", func(t *testing.T) {
```

Comment:
I expect the `GetBackplaneConfiguration()` should succeed if the proxy-url is not configured. I am also confused on why this test succeed.....